src/aleph/http.clj

```diff
@@ -45,7 +45,7 @@
    | `max-initial-line-length` | the maximum characters that can be in the initial line of the request, defaults to `8192`
    | `max-header-size`         | the maximum characters that can be in a single header entry of a request, defaults to `8192`
    | `max-chunk-size`          | the maximum characters that can be in a single chunk of a streamed request, defaults to `16384`
-   | `epoll?`                  | if `true`, uses `epoll`, defaults to `false`.
+   | `transport`               | the transport to use, one of `:epoll`, `:kqueue` or `:io-uring`
```

Comment:
Mention which one it defaults to, if unspecified

Reply:
> :nio should be added for completeness, and we should mention that :transport defaults to it, if unspecified.

Agreed, I will modify the docstrings accordingly.

> Maybe we should leave in the epoll? option, but deprecate it, and say that :transport supersedes it?

I would remove the `epoll?` option, that way, no-one will ever use it in the future.